docs/kcMigration/KeycloakMigration.md

```diff
@@ -0,0 +1,50 @@
+---
+description: KeyCloak (RH-SSO) Migration from Pathfinder to BCGov service
+resourceType: Documentation
+title: BC Government KeyCloak (RH-SSO) Service Migration Plan
+---
+
+## KeyCloak Service Migration
+
+Hi team, we are in the process of transforming KeyCloak from a pathfinder service to an organizational BCGov service. For more information on this Single Sign-On service, refer to [here](https://developer.gov.bc.ca/BC-Government-SSO-Service-Definition).
+
+Part of the migration requires you (application teams using this existing SSO) to update the KeyCloak configurations. As for now, there is no immediate actions to migrate. We will inform the community in [RocketChat #sso channel](https://chat.pathfinder.gov.bc.ca/channel/sso) as well as product owners when ready so that you could plan ahead for some sprint time to work on the relevant tasks.
+
+At the moment, we are collecting application team information for each KC realm. Please visit the rocketchat discussion [here](to-be-generated), find your realm and provide the details listed:
+- Project Name: 
+- KC realm ID: 
+- OpenShift project set/s: 
+
+If your team is not on the OpenShift platform, please provide contact information for:
+- Product owner: 
+- Technical lead: 
+
+
```

Comment:
```suggestion
At this point, there are no immediate actions required to migrate.  Once the new service name(s) are in place, you will be required to update your realm's KeyCloak configurations as well as any application links to the keycloak DNS.  We will inform the community in [RocketChat #sso channel](https://chat.pathfinder.gov.bc.ca/channel/sso) as well as product owners as the start date approaches so that you can plan ahead for some sprint time to work on the relevant tasks.

```
To give the teams the heads up of more work to come.